digicampipe/utils/utils.py

```diff
@@ -1,9 +1,28 @@
 import numpy as np
 import scipy.ndimage as ndimage
-
+from scipy.interpolate import interp1d
 # Define the integration function
 
 
+def filter_template(filename_pulse_shape, dt):
```

Comment:
I do not understand this function name without reading the implementation:
 * is this going to filter some template, as in "I am filtering the template"
 * or is this a kind of getter, which gives me a filter-template, i.e. a template for a filter

Reply:
This might not end up in PR. I was using different pulse finding algorithm. One that sticked out was : 
find_peak_cwt from scipy (continous wavelet transform.) I was just trying out if I can make a wavelet from the pulse_shape (don't know yet if this is the right approach...). Will work on this soon